deploy/runtools/run_farm_deploy_managers.py

```diff
@@ -10,6 +10,9 @@
 from fabric.contrib.project import rsync_project # type: ignore
 import time
 from os.path import join as pjoin
+from os import PathLike, fspath
+from fsspec.core import url_to_fs # type: ignore
```

Comment:
Looks like fsspec doesn't have type annotations or hints available yet but they aren't opposed to discussing them if this becomes problematic. https://github.com/fsspec/filesystem_spec/issues/625